_posts/2021-02-03-new-ansible-module-copr.md

```diff
@@ -0,0 +1,66 @@
+
+## Module usage
+
+To use it in a playbook, specify: `community.general.copr`.
+
+### Examples of use in a playbook 
+
+Use state enabled to enable the copr project, then use the name of the user/project and the chroot you wish to enable. 
+
+    - name: Enable project Test of the user schlupov
+      community.general.copr:
+        host: copr.fedorainfracloud.org
+        state: enabled
+        name: schlupov/Test
+        chroot: fedora-31-x86_64
+
+Use state absent to remove the copr project. The host is optional and copr.fedorainfracloud.org is used as the default. Likewise, chroot is not required, so if you use the module on fedora 32 and the x86_64 architecture, fedora-32-x86_64 will be used as chroot option.
+
+    - name: Remove project copr of the group copr
+      community.general.copr:
+        state: absent
+        name: @copr/copr
+
+Use state disabled to disable the copr project. The repository can be seen in the /etc/yum.repos.d/ directory, but is set to disabled. On the other hand, in case of state absent, the repo file is completely removed from the system.
+
+    - name: Disable project copr of the group copr
+      community.general.copr:
+        state: disabled
+        name: @copr/copr
+        chroot: fedora-32-x86_64
+
+## Parameters
+
+| Parameter |  Choices/Defaults                                            | Comments                                                                                                                                                                                                                                    | Required |
+|-----------|--------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| host      | Default: "copr.fedorainfracloud.org"                         | The Copr host to work with                                                                                                                                                                                                                  | No       |
+| protocol  | Default: "https"                                             | This indicate which protocol to use with the host                                                                                                                                                                                           | No       |
+| name      |                                                              | Copr directory name, for example @copr/copr-dev                                                                                                                                                                                             | Yes      |
```

Comment:
s/directory/project/

Reply:
Directory is fine IMO because that's how we call the `@copr/copr:pr:19` thingy.

Reply:
I decided that the word project is better.